pages/en/lb2/readmes/loopback-example-connector.md

```diff
@@ -0,0 +1,33 @@
+# loopback-example-connector.soap
```

Comment:
Figured out part of the problem.. the loopback-example-XXX repos that are using branches to differentiate between modules were all clobbering each other because the readme name was being based only on the repo name.